README.md

```diff
@@ -1 +1,35 @@
-# Milestone1
\ No newline at end of file
+# Milestone1
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> A responsive flexbox webpage for a concert company, with Doom manipulating objects JavaScript, HTML, CSS, and flexbox.
+
+## Online Webpage
+- https://github.com/renercrows/Milestone1/tree/main
+
+## Built With
+
+- HTML, CSS, Flexbox, Media Query and, DOOM
+
+## Author
+
+👤 **Maximiliano Ungredda**
+
+- GitHub: [@renercrows](https://github.com/renercrows)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).